src/geode/basic/cell_array.cpp

```diff
@@ -33,7 +33,7 @@ namespace geode
     {
     public:
         Impl() = default;
-        Impl( std::array< index_t, dimension > cells_number )
+        explicit Impl( std::array< index_t, dimension > cells_number )

```

Comment:
warning: constructor does not initialize these fields: cells_number_ [cppcoreguidelines-pro-type-member-init]
```cpp
ublic:
                                           ^
```